libraries/AP_RCTelemetry/AP_CRSF_Telem.cpp

```diff
@@ -396,9 +405,13 @@ bool AP_CRSF_Telem::is_packet_ready(uint8_t idx, bool queue_empty)
         return false;
 #endif
     case PASSTHROUGH:
+    #if AP_RC_CHANNEL_ENABLED
```

Comment:
This now falls through when it shouldn't? 

Similarly elsewhere

Reply:
fixed in multiple places